conf/test_assembly_input.config

```diff
@@ -20,21 +20,24 @@ process {
 }
 
 params {
-    config_profile_name           = 'Test profile'
-    config_profile_description    = 'Minimal test dataset to check pipeline function'
+    config_profile_name           = 'Test assembly input profile'
+    config_profile_description    = 'Minimal test dataset to check assembly input'
 
-    // Input data
     input                         = params.pipelines_testdata_base_path + 'mag/samplesheets/samplesheet.v4.csv'
     assembly_input                = params.pipelines_testdata_base_path + 'mag/samplesheets/assembly_samplesheet.csv'
-    min_length_unbinned_contigs   = 1
-    max_unbinned_contigs          = 2
+
     busco_db                      = params.pipelines_testdata_base_path + 'mag/databases/busco/bacteria_odb10.2024-01-08.tar.gz'
     busco_db_lineage              = 'bacteria_odb10'
-    skip_gtdbtk                   = true
-    gtdbtk_min_completeness       = 0.01
+
+    skip_quast                    = true
+    skip_prodigal                 = true
+    skip_prokka                   = true
+    skip_concoct                  = false
     refine_bins_dastool           = true
-    refine_bins_dastool_threshold = 0
-    // TODO not using 'both' until #489 merged
-    postbinning_input             = 'refined_bins_only'
-    busco_clean                   = true
+    refine_bins_dastool_threshold = 0.0
+    postbinning_input             = 'both'
+    // run_gunc                   = true
```

Comment:
Why si this deactiated atm? It doesn't fit?

Reply:
Sorry, I didn't put a WIP there.
Fill GUNC db doesn't fit. I was planning to build a mini db following your attempt here: https://github.com/nf-core/mag/pull/621#issuecomment-2326045689

Reply:
OK nice!